### DISCUSSION TITLE
Use dark mode for UI

### DISCUSSION THREAD

**Discussion by g4rcez** [GhostManager/Ghostwriter#277] [Ideas]
Dark mode it's an excellent feature to have in any product. It has many features, as you can see in [this link](https://www.venturus.org.br/en/darkmode-and-the-benefits-for-the-user-experience/).

I am able to implement this feature in the project, but we need to discuss it first.

**Comment by chrismaddalena** [comment]
That's something we can discuss. I've looked into the possibility of adding a dark mode toggle, but it's remained a low priority until other requested features are complete.

**Comment by lucasgates** [comment]
Would love to see dark mode.
